internal/collector/fluentd/visitors.go

```diff
@@ -33,6 +35,7 @@ func CollectorVisitor(collectorContainer *v1.Container, podSpec *v1.PodSpec) {
 			},
 		},
 		v1.EnvVar{Name: "RUBY_GC_HEAP_OLDOBJECT_LIMIT_FACTOR", Value: "0.9"},
+		v1.EnvVar{Name: "OPENSSL_CONF", Value: fmt.Sprintf("%s/openssl.cnf", configVolumePath)},
```

Comment:
Can we add it once here? https://github.com/openshift/cluster-logging-operator/blob/9fb2003d041c3464c213294ef4ae8faf8dd56710/internal/collector/collector.go#L183

Reply:
no, the path are collector type specific
fluentd uses `/etc/fluent/configs.d/user`
vector uses `/etc/vector`